doc/user_guide/user_guide.rst

```diff
@@ -69,20 +69,24 @@ Clone the repository
 
    git clone https://github.com/exasol/integration-test-docker-environment
```

Comment:
We can also remove the cloning of the repo, this is not needed anymore. We should replace it with the explanation for the executable.